tst/unit/test_mesh_data.cpp

```diff
@@ -101,6 +104,15 @@ TEST_CASE("MeshData works as expected for simple packs", "[MeshData]") {
                 }
               });
         }
+        auto &var = pmbd->Get("v6");
+        par_for(
+            loop_pattern_mdrange_tag, "initialize v6", DevExecSpace(), kb.s, kb.e + 1,
+            jb.s, jb.e + 1, ib.s, ib.e + 1, KOKKOS_LAMBDA(int k, int j, int i) {
+              Real n = i + 1e1 * j + 1e2 * k + 1e3 * 0 + 1e4 * 4 + 1e5 * b;
+              if (k <= kb.e && j <= jb.e) var(0, 0, 0, 0, k, j, i) = 3 * n + 0;
+              if (k <= kb.e && i <= ib.e) var(1, 0, 0, 0, k, j, i) = 3 * n + 1;
+              if (j <= jb.e && i <= ib.e) var(2, 0, 0, 0, k, j, i) = 3 * n + 2;
```

Comment:
I think you would never want to do this in practice, just for testing. This is just to fill a variable with values without using a pack.

Reply:
Yeah I suppose we always interact with things through packs, so maybe it doesn't matter. I guess I just wonder if instead of setting the slowest-possible index to be the topological location, we could set it to the slowest one in practice? Maybe that's not what we want?

Reply:
I think that necessitates extra logic in a number of places in the code (e.g. boundaries), so I would avoid changing the dimension of the topological index unless absolutely necessary.